src/flisp/cvalues.c

```diff
@@ -4,28 +4,31 @@
 #define NWORDS(sz) (((sz)+3)>>2)
 #endif
 
-static int ALIGN2, ALIGN4, ALIGN8, ALIGNPTR;
-
-value_t int8sym, uint8sym, int16sym, uint16sym, int32sym, uint32sym;
-value_t int64sym, uint64sym;
-value_t ptrdiffsym, sizesym, bytesym, wcharsym;
-value_t floatsym, doublesym;
-value_t gftypesym, stringtypesym, wcstringtypesym;
```

Comment:
This (`gftypesym`) should be the only global symbol that I removed. It doesn't seem to be used anywhere. @JeffBezanson